include/erlcloud_ddb2.hrl

```diff
@@ -207,4 +207,106 @@
         {time_to_live_specification :: undefined | #ddb2_time_to_live_specification{}
         }).
 
+-record(ddb2_backup_summaries,
+        {backup_arn:: undefined | binary(),
+         backup_creation_date_time :: undefined | date_time(),
+         backup_name:: undefined | binary(),
+         backup_size_bytes :: undefined | pos_integer(),
+         backup_status:: undefined | binary(),
+         table_arn:: undefined | binary(),
+         table_id:: undefined | binary(),
+         table_name:: undefined | binary()
+        }).
+
+
+-record(ddb2_list_backups,
+        {backup_summaries :: undefined | [#ddb2_backup_summaries{}],
+         last_evaluated_backup_arn :: undefined | erlcloud_ddb2:table_name()
+        }).
+
+-record(ddb2_backup_details,
+        {backup_arn:: undefined | binary(),
+         backup_creation_date_time :: undefined | date_time(),
+         backup_name:: undefined | binary(),
+         backup_size_Bytes :: undefined | pos_integer(),
+         backup_status:: undefined | binary()
+        }).
+
+
+-record(ddb2_create_backup,
+        {backup_details :: undefined | #ddb2_backup_details{}
+        }).
+
```

Comment:
I know there are some consistency issues in the header regarding whitespace between records, but can we keep it to one empty line? Feel free to add whitespace between the records with none—or not.

Reply:
This still isn't addressed.

Reply:
Sorry, i can't understand... Where i must add whitespace ?

Reply:
This one is right, I misread the diff. Sorry about that.